src/ConfigurationException/InvalidConfigurationException.php

```diff
@@ -20,6 +20,7 @@
  * @author SpacePossum
  *
  * @internal
+ * @final Only internal extending this class is supported
```

Comment:
i must admit this looks weird.

if it's not final we know we could extend it, and we do know it's possible only internally as `@internal` is there, so any contact with this class is internal

Reply:
class is not final. annotation is wrong.

Reply:
yeah, it is basically this case https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1389#issuecomment-139918457 :)
Guess we've no good answer now. We could add some other tag to this class and put that in the black list though.